### ISSUE TITLE
SPICA: Interactive Video Content Exploration through Augmented Audio Descriptions for Blind or Low-Vision Viewers

### ISSUE THREAD

**Issue by Masa630** [SeitaKayukawa/Blind_Accessibility_Papers#608]
### Links
- Paper: https://arxiv.org/pdf/2402.07300v2

### Abstract
Blind or Low-Vision (BLV) users often rely on audio descriptions (AD) to access video content. However, conventional static ADs can leave out detailed information in videos, impose a high mental load, neglect the diverse needs and preferences of BLV users, and lack immersion. To tackle these challenges, we introduce Spica, an AI-powered system that enables BLV users to interactively explore video content. Informed by prior empirical studies on BLV video consumption, Spica offers interactive mechanisms for supporting temporal navigation of frame captions and spatial exploration of objects within key frames. Leveraging an audio-visual machine learning pipeline, Spica augments existing ADs by adding interactivity, spatial sound effects, and individual object descriptions without requiring additional human annotation. Through a user study with 14 BLV participants, we evaluated the usability and usefulness of Spica and explored user behaviors, preferences, and mental models when interacting with augmented ADs.

・視覚障害者がビデオコンテンツをインタラクティブに探索できるシステムSPICA
・ビデオの時間的なナビゲーションとフレーム内の空間探索を可能にする
<img width="862" alt="スクリーンショット 2024-06-06 19 29 53" src="https://github.com/SeitaKayukawa/Blind_Accessibility_Papers/assets/127022571/6d33e395-c0e2-4b8b-8d30-c40d311a3dc8">

<img width="856" alt="スクリーンショット 2024-06-06 19 32 33" src="https://github.com/SeitaKayukawa/Blind_Accessibility_Papers/assets/127022571/9aeec7e2-7055-465a-8b2d-3e67029c7025">

・フレームごとのcaptionを生成して、映像を止めてcaptionを聞くことができる。逆にcaptionから自分が興味のあるシーンを探索することができる。
・また、フレーム内にどんな物体が存在するかをタップまたはリストから知ることができる。

・ユーザ実験では視覚障害者の没入感を高めた。
F1: Original ADs have an anchor effect on users’ active video pausing and frame exploration.
F2: Users pause or temporarily explore the video for reflecting, validating, and customizing their content consumption experience.
F3: The granularity and quality of the frame-level description affect users’ decisions on whether to access object information further.
F4: Users often explore individual objects when they hear new sounds from the original video or they hear particular objects mentioned in the ADs.
